src/models/games-history/services/ptn.service.spec.ts

```diff
@@ -76,10 +76,12 @@ describe('PTNService', () => {
 			const game2 = { id: 500121, date: 1461430700000, size: 7, player_white: 'Covault', player_black: 'Catalyst', notation: 'P A7,P D4,P E4,P A4,P D3,P B4,P D2,P D7,P D5,P D6,P D1,P C4 C,P E5,M C4 D4 1,P E6,P E7,P F6,P F7,P G7 C,M D4 E4 2,M G7 F7 1,M E7 E6 1,M F6 E6 1,M E4 E5 3,M E6 C6 1 2,P D4 C,P C7,P C4,P C5,P G4,P E4 C,P A5,P C3,M E5 D5 4,M C3 C4 1,M D5 C5 5,M C7 D7 1,M C5 C6 6,P E5,M C6 D6 7,M F7 E7 2,M D4 C4 1,P E3,M D6 E6 7,P D5', result: 'R-0', timertime: 180, timerinc: 20, rating_white: 1080, rating_black: 1105, unrated: 0, tournament: 0, komi: 0, pieces: -1, capstones: -1, rating_change_white: 164, rating_change_black: -66};
-			const ptn1 = `[Site "PlayTak.com"]\n[Event "Online Play"]\n[Date "2022.05.25"]\n[Time "14:19:10"]\n[Player1 "Covault"]\n[Player2 "Catalyst"]\n[Clock "3:0 +20"]\n[Result "R-0"]\n[Size "7"]\n[Komi "0"]\n[Flats "40"]\n[Caps "2"]\n\n\n1. a7 d4\n2. e4 a4\n3. d3 b4\n4. d2 d7\n5. d5 d6\n6. d1 Cc4\n7. e5 1c4>1\n8. e6 e7\n9. f6 f7\n10. Cg7 2d4>2\n11. 1g7<1 1e7-1\n12. 1f6<1 3e4+3\n13. 3e6<12 Cd4\n14. c7 c4\n15. c5 g4\n16. Ce4 a5\n17. c3 4e5<4\n18. 1c3+1 5d5<5\n19. 1c7>1 6c5+6\n20. e5 7c6>7\n21. 2f7<2 1d4<1\n22. e3 7d6>7\n23. d5\nR-0\n`;
-			const ptn2 = `[Site "PlayTak.com"]\n[Event "Online Play"]\n[Date "2016.04.23"]\n[Time "16:58:20"]\n[Player1 "Anon"]\n[Player2 "Anon"]\n[Clock "3:0 +20"]\n[Result "R-0"]\n[Size "7"]\n[Komi "0"]\n[Flats "40"]\n[Caps "2"]\n\n\n1. a7 d4\n2. e4 a4\n3. d3 b4\n4. d2 d7\n5. d5 d6\n6. d1 Cc4\n7. e5 1c4>1\n8. e6 e7\n9. f6 f7\n10. Cg7 2d4>2\n11. 1g7<1 1e7-1\n12. 1f6<1 3e4+3\n13. 3e6<12 Cd4\n14. c7 c4\n15. c5 g4\n16. Ce4 a5\n17. c3 4e5<4\n18. 1c3+1 5d5<5\n19. 1c7>1 6c5+6\n20. e5 7c6>7\n21. 2f7<2 1d4<1\n22. e3 7d6>7\n23. d5\nR-0\n`;
+			const ptn1 = `[Site "PlayTak.com"]\n[Event "Online Play"]\n[Date "2022.05.25"]\n[Time "14:19:10"]\n[Player1 "Covault"]\n[Rating1 "1080"]\n[Player2 "Catalyst"]\n[Rating2 "1105"]\n[Clock "3:0 +20"]\n[Result "R-0"]\n[Size "7"]\n[Komi "0"]\n[Flats "40"]\n[Caps "2"]\n\n\n1. a7 d4\n2. e4 a4\n3. d3 b4\n4. d2 d7\n5. d5 d6\n6. d1 Cc4\n7. e5 1c4>1\n8. e6 e7\n9. f6 f7\n10. Cg7 2d4>2\n11. 1g7<1 1e7-1\n12. 1f6<1 3e4+3\n13. 3e6<12 Cd4\n14. c7 c4\n15. c5 g4\n16. Ce4 a5\n17. c3 4e5<4\n18. 1c3+1 5d5<5\n19. 1c7>1 6c5+6\n20. e5 7c6>7\n21. 2f7<2 1d4<1\n22. e3 7d6>7\n23. d5\nR-0\n`;
+			const ptn2 = `[Site "PlayTak.com"]\n[Event "Online Play"]\n[Date "2016.04.23"]\n[Time "16:58:20"]\n[Player1 "Anon"]\n[Rating1 "1080"]\n[Player2 "Anon"]\n[Rating2 "1105"]\n[Clock "3:0 +20"]\n[Result "R-0"]\n[Size "7"]\n[Komi "0"]\n[Flats "40"]\n[Caps "2"]\n\n\n1. a7 d4\n2. e4 a4\n3. d3 b4\n4. d2 d7\n5. d5 d6\n6. d1 Cc4\n7. e5 1c4>1\n8. e6 e7\n9. f6 f7\n10. Cg7 2d4>2\n11. 1g7<1 1e7-1\n12. 1f6<1 3e4+3\n13. 3e6<12 Cd4\n14. c7 c4\n15. c5 g4\n16. Ce4 a5\n17. c3 4e5<4\n18. 1c3+1 5d5<5\n19. 1c7>1 6c5+6\n20. e5 7c6>7\n21. 2f7<2 1d4<1\n22. e3 7d6>7\n23. d5\nR-0\n`;
 			const getPTN1 = service.getPTN(game1);
+			console.log(getPTN1);
```

Comment:
Should these stay?

Reply:
These are unit tests, so it should be fine.

Reply:
They are unneeded removing